container-engine-lib/lib/backend_impls/docker/docker_manager/docker_manager.go

```diff
@@ -1433,7 +1434,7 @@ func (manager *DockerManager) BuildImage(ctx context.Context, imageName string,
 		CgroupParent:   "",
 		NetworkMode:    "",
 		ShmSize:        0,
-		Dockerfile:     defaultContainerImageFile,
+		Dockerfile:     buildFile,
```

Comment:
This was missing in the previous implementation

Reply:
ah I missed this as well, thanks for this fix